packages/table/src/utils/createTableItems.spec.ts

```diff
@@ -0,0 +1,217 @@
+import { DataStream, Viewport } from '@iot-app-kit/core';
+import { Annotations, COMPARISON_OPERATOR, getThresholds } from '@synchro-charts/core';
+import { createTableItems } from './createTableItems';
+
+const dataStreams: DataStream[] = [
+  {
+    id: 'data-1',
+    data: [
+      { y: 0, x: new Date(2021, 1, 1, 0, 0, 1).getTime() },
+      { y: 1, x: new Date(2022, 1, 1, 0, 0, 2).getTime() },
+      { y: 2, x: new Date(2022, 1, 1, 0, 0, 3).getTime() },
+      { y: 3, x: new Date(2022, 1, 1, 0, 0, 4).getTime() },
+      { y: 4, x: new Date(2022, 1, 1, 0, 0, 5).getTime() },
+    ],
+    resolution: 0,
+  },
+  {
+    id: 'data-2',
+    data: [{ y: 11, x: new Date(2022, 1, 1, 0, 0, 1).getTime() }],
+    resolution: 0,
+  },
+  {
+    id: 'agg_data',
+    aggregates: {
+      60: [{ y: 60, x: new Date(2022, 1, 1, 0, 0, 1).getTime() }],
+    },
+    data: [{ y: 0, x: new Date(2022, 1, 1, 0, 0, 1).getTime() }],
+    resolution: 60,
+  },
+];
+
+const viewport = {
+  duration: '1000',
+};
+
+const itemWithRef = [
+  {
+    value1: {
+      $cellRef: {
+        id: 'data-1',
+        resolution: 0,
+      },
+      noRef: 'No Ref',
+    },
+    value2: {
+      $cellRef: {
+        id: 'data-2',
+        resolution: 0,
+      },
+    },
+    noRef: {
+      data: [1, 2, 3, 4],
+    },
+    rawValue: 10,
+  },
+  {
+    data: {
+      $cellRef: {
+        id: 'data-1',
+        resolution: 0,
+      },
+    },
+    invalid: {
+      $cellRef: {
+        id: 'invalid-data-stream',
+        resolution: 0,
+      },
+    },
+    aggregates: {
+      $cellRef: {
+        id: 'agg_data',
+        resolution: 60,
+      },
+    },
+    invalidAggregation: {
+      $cellRef: {
+        id: 'agg_data',
+        resolution: 55,
+      },
+    },
+  },
+];
+
+describe('createTableItems', () => {
```

Comment:
still have unnecessary describe blocks

Reply:
I see what do you mean now. Removing all describe blocks.